lib/internal/navigator.js

```diff
@@ -104,7 +104,9 @@ class Navigator {
    * @return {string}
    */
   get language() {
-    return language;
+    // The default locale might be changed dynamically, so always invoke the
+    // binding.
+    return getDefaultLocale() || 'en-US';
```

Comment:
I don't know if you want to handle that here but it's possible for the locale to be "und" (for "undefined")

Reply:
Presumably if someone sets the locale explicitly to `und`, they'd want to test with `und` instead of `en-US`, so this might be ok as-is?

Reply:
That is computed by `v8::internal::Isolate::DefaultLocale()` and consistent with `Intl?.Collator().resolvedOptions().locale` for bogus locales, IMO that makes sense, we should just return whatever that's considered the default locale by the Intl implementation, in case someone tries to pipe it into Intl and wants to get consistent results. (On a side note according to [BCP 47](https://www.rfc-editor.org/rfc/bcp/bcp47.txt) which references ISO 639-2 it's "undetermined")

Reply:
Is the default language mutable at runtime here? Just thinking that it would likely make sense to cache the value in a private field if the value is expected to remain fixed so that we don't call `getDefaultLocale()` on every access of `navigator.language`

Reply:
It can be mutated by an addon, which is what `v8::Isolate::LocaleConfigurationChangeNotification()` is for.

Reply:
Got it. Still tho, I wonder which is going to be more common, repeat accesses to `navigator.language` or mutating the value. Maybe it can be cached by if the locale is changed we can signal to refresh the cached value? In any case, doesn't need to be done here.